Create Slack and GitHub Connector addons (#115)
kyma-incubator/hack-showcase: addons/githubconnector-1.0.0/chart/githubconnector/README.md

# Github Connector Chart<!-- omit in toc -->

## Overview
The chart creates the GitHub Connector Deployment and a Namespace in Kyma.

> Additionally, it creates a service, binds it to the newly created Namespace and exposes its API. Apart from that, it creates an Application.

Reviewer: ```suggestion
Additionaly it creates an Application, registers a service, binds it to the newly created Namespace and exposes its API.
```
